doc/script_commands.txt

```diff
@@ -1374,6 +1375,24 @@ for more details.
 
 ---------------------------------------
 
+*zmes1("<string>")
```

Comment:
Can you also add a mention of the `zmes1f()`/`zmes2f()` variants, so that someone searching this file for them will find a match?